### PR TITLE
Added the use of the elastic stopword capabilities to the variable an…

### PR DESCRIPTION
**Pull request by HowardLander**
…d concept indices. Also fixed the cli.py command search command to return actual json

### CODE REVIEW

File: src/dug/cli.py [hunk 1]
```diff
@@ -105,12 +106,11 @@ def search(args):
...
     # dug = Dug()
     response = dug.search(args.target, args.query, **args.kwargs)
-
     if not response:
        print ("No search results found")
     else:   
-       print(response)
-
+       jsonResponse = json.dumps(response, indent = 2)
+       print(jsonResponse)
```
Comment (cschreep) [inline src/dug/cli.py:113]: This works as expected. However I notice that if there are no results, then the code on line 110 doesn't get executed. It returns
```{
  "total_items": 0
}
```

**Review by cschreep — APPROVED** [review]
This appears to work as expected. I believe that this will require anyone developing locally to tear down any existing ES index so that the code will recreate it with the new config, is that correct?

**Comment by cschreep** [comment]
There is a failure in the test_cli that needs updating

**Comment by HowardLander** [comment]
Yes.

________________________________

**Comment by HowardLander** [comment]
Huh,  That's odd...

________________________________

**Pull request merged**